frontend/src/pages/home.tsx

```diff
@@ -113,19 +113,22 @@ const Home: FC = () => {
                 </TabList>
               )}
               {extraSmall && (
-                <div className="px-4 py-2">
+                <div className="">
                   <div className="flex gap-1">
+                    <div className='border-b-2 w-1/2 text-center border-primary-700 p-2'>
+                      <Button
+                        variant="text"
+                        onClick={() => setValue('learn')}
```

Comment:
Is it normal that onClick you set the current tabl to `learn` even if the text is different?

Reply:
No, not sure why it's there to begin with I removed it